src/components/hocs/withIntersection.js

```diff
@@ -31,19 +31,25 @@ export default (Component, { Fallback: DefaultFallback = Fallback } = {}) => {
         padding,
         "data-chartid": dataChartId,
         "data-testid": dataTestId = "chartIntersector",
+        parentRef,
         ...rest
       },
-      parentRef
+      ref
     ) => {
       const chart = useChart()
-      const [ref, setRef] = useForwardRef(parentRef)
+      const [myRef, setRef] = useForwardRef(ref)
```

Comment:
I think it would be better to also update `setRef` to `setMyRef` in order to be clear which value is updated! 😄